corehq/apps/analytics/tasks.py

```diff
@@ -193,6 +194,20 @@ def track_clicked_deploy_on_hubspot(webuser, cookies, meta):
     _send_form_to_hubspot(HUBSPOT_CLICKED_DEPLOY_FORM_ID, webuser, cookies, meta)
 
 
+@task(queue="background_queue", acks_late=True, ignore_result=True)
+def track_user_subscriptions_on_hubspot(webuser, properties):
+    vid = _get_user_hubspot_id(webuser)
+    if vid:
```

Comment:
curious why you need this? I thought hubspot automatically creates a contact if it doesn't exist?